buda/entities/visit.proto

```diff
@@ -40,4 +40,4 @@ message Visit {
     repeated Experiment experiments = 11;
     string source = 12;
     Product product = 13;
-}
```

Comment:
Hey @mallen5311! I think there might be a small syntax error here, the closing brace on the message is missing? Otherwise it seems good!

Reply:
Good catch!  Weird though, the closing brace on the message seems to be in my local copy, but it's not here in the PR (and no pending changes or commits on the branch) 🤷‍♂️ .  I'll get it fixed up though 😄